frontend/public/style/_overrides.scss

```diff
@@ -358,6 +147,23 @@ kbd {
   }
 }
 
+// specificity targeting form elements to override --pf-global--FontSize--md
```

Comment:
What impact does this have if we change this globally to `$font-size-base` as we transition to PF4? Does it break the nav/masthead? The dev console team is looking at bringing in some other PF4 components and will have similar font-size issues.

cc @rhamilto @christianvogt

Reply:
It would be nice if we can override `--pf-global--FontSize--md` and set it to 14px so that every new PF4 component we utilize doesn't need a separate font tweak.

Reply:
@spadgett Setting `--pf-global--FontSize--md` to `$font-size-base: 14px` globally still requires a few targeted overrides. Can we make those global font-size changes as a follow-on and go forward with merging this pf, so the font related changes can be easier tracked?

Reply:
@sg00dwin I was going to suggest a separate PR, yeah. Then we can get that merged separately to unblock the dev console team. It looks like you have some test failures to work through still in this one.